controllers/util/cluster.go

```diff
@@ -0,0 +1,63 @@
+package util
```

Comment:
Hi @yastij,

I'm not sure it's worth introducing a new package for this one function. Unless there's a strong reason for this, let's just fold this back into the `controllers` package as a private function.